owncloud/core
# Ensure the /user/files directory is always present in the FS

## Description
Ensure the user's folder always exists. In addition, update the login timestamp after the user is prepared to login, in particular, after the skeleton files have been copied.

## Related Issue
https://github.com/owncloud/enterprise/issues/5187

## Motivation and Context
There might be temporary errors preventing the user's folder from being created properly. If this happens, the admin would need to create the directory by himself. This PR ensure that the user's folder is eventually created, assuming it's a temporary error.

## How Has This Been Tested?
Manually tested.
1. Login with a user
2. Remove the "/user/files" directory from the command line
3. Login again with the same user

The "/user/files" directory is recreated.
Note that step 2 is mostly for testing. The real expectation is that something goes wrong during the log in in step 1 and the "/user/files" directory isn't created in the first place.

## Screenshots (if appropriate):

## Types of changes


## Checklist:
